packit_service/worker/jobs.py

```diff
@@ -283,6 +287,17 @@ def report_task_accepted(
             task_accepted_time, handler_kls, number_of_build_targets
         )
 
+    def search_distgit_config_in_issue(self):
+        issue = self.event.project.get_issue(self.event.issue_id)
+        if m := match(r"[\w\s-]+dist-git \((\S+)\):", issue.description):
```

Comment:
could you please add a comment here on what is the format of the description where we are checking the match?